osbuild/util/ostree.py

```diff
@@ -8,6 +8,7 @@
 import typing
 # pylint doesn't understand the string-annotation below
 from typing import Any, List  # pylint: disable=unused-import
+from glob import glob
```

Comment:
optional: I would just `import glob` and then use `glob.glob()` in the code below.